aws/data_source_aws_api_gateway_rest_api.go

```diff
@@ -21,6 +23,54 @@ func dataSourceAwsApiGatewayRestApi() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"policy": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"api_key_source": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"minimum_compression_size": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"binary_media_types": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"endpoint_configuration": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"types": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
+						"vpc_endpoint_ids": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
+					},
+				},
+			},
+			"execution_arn": {
```

Comment:
New attribute is missing documentation, will add on merge.

Reply:
Sorry about that, this happened to me more than once and got me thinking if we can have some sort of validation for this with a linter. ill take a look if i can hack something.

Reply:
Its partially possible today, but requires the output of `terraform providers schema -json`, which is not PR friendly without some Terraform CLI and plugin directory setup in CI. We're hoping to allow the provider binary to output its own form of that JSON instead (with all data; Terraform core does not receive the full schema information from providers) to remove the Terraform CLI and ancillary setup, but that's likely a few weeks out.

I have some centralized tooling issues available here (https://github.com/bflad/tfproviderdocs/issues/15 / https://github.com/bflad/tfproviderdocs/issues/16), but we'd likely also want Markdown AST parsing in that tooling (rather than regular expressions, for example) before implementing a lot of the Markdown checking to prevent false positives.